package.json

```diff
@@ -108,6 +108,7 @@
     "concurrently": "6.4.0",
     "conventional-changelog-cli": "2.1.1",
     "cpy-cli": "^3.1.1",
+    "cssnano": "5.0.12",
```

Comment:
added this because the build was encountering a CSSSyntaxError. https://tailwindcss.com/docs/optimizing-for-production

Reply:
I think it was erroring due to comments in CSS.

Reply:
removed. not needed.